org.alloytools.alloy.application/src/main/java/edu/mit/csail/sdg/alloy4viz/AlloyType.java

```diff
@@ -102,6 +106,13 @@ public AlloyType(String name, boolean isOne, boolean isAbstract, boolean isBuilt
      */
     public final boolean isEnum;
 
+    /**
+     * Records whether this sig is known to be "var"; NOTE: this value is NOT USED
+     * during equals() comparison.
```

Comment:
maybe add a note about why this is ignored in `equals()`

Reply:
I was following the established behaviour, other attributes are also not considered